packages/vuetify/src/components/VDatePicker/util/index.ts

```diff
@@ -1,9 +1,11 @@
 import createNativeLocaleFormatter from './createNativeLocaleFormatter'
 import monthChange from './monthChange'
 import pad from './pad'
+import determineWeeknumber from './determineWeeknumber'
```

Comment:
capitalize number (`determineWeekNumber`), here and in numerous other places

Reply:
I moved the content of this file to the new dateTimeUtils.js file in the global util directory and renamed it to weekNumber.